### PR TITLE
rewrite model section

### PR DESCRIPTION
**Pull request by fluffy**

### CODE REVIEW

File: draft-lcurley-moq-transport.md [hunk 1]
```diff
@@ -167,36 +167,32 @@ x (b):
...
-The basic element of MoQTransport is an *object*.
+The basic data element of MoQTransport is an *object*.
 An object is an addressable unit whose payload is a sequence of bytes.
 All objects belong to a group, indicating ordering and potential dependencies. {{model-group}}
-Objects carry associated metadata such as priority, TTL, or other information usable by a relay, but relays MUST treat the object payload as opaque.
-
-The application is solely responsible for the contents of objects.
-This includes the underlying encoding, compression, any end-to-end encryption, or authentication.
-A relay MUST NOT combine, split, or otherwise modify object payloads.
+Objects are comprised of two parts: metadata and a payload.  The metadata is visible to relays but the payload is only visible to the producer and consumer. The application is solely responsible for the content of the object payload.
```
Comment (fluffy) [inline draft-lcurley-moq-transport.md:177]: I would rather not loose priority and TTL in metadata here.

File: draft-lcurley-moq-transport.md [hunk 2]
```diff
@@ -167,36 +167,32 @@ x (b):
...
 ## Objects {#model-object}
 
-The basic element of MoQTransport is an *object*.
+The basic data element of MoQTransport is an *object*.
 An object is an addressable unit whose payload is a sequence of bytes.
 All objects belong to a group, indicating ordering and potential dependencies. {{model-group}}
-Objects carry associated metadata such as priority, TTL, or other information usable by a relay, but relays MUST treat the object payload as opaque.
-
-The application is solely responsible for the contents of objects.
-This includes the underlying encoding, compression, any end-to-end encryption, or authentication.
```
Comment (kixelated) [inline draft-lcurley-moq-transport.md:180]: Could we mention that the application is responsible for any further sub divisions?

File: draft-lcurley-moq-transport.md [hunk 3]
```diff
@@ -167,36 +167,32 @@ x (b):
 : Indicates that x consists of a variable length integer, followed by that many bytes of binary data.
 
 
-# Model
+# Object Model {#model}
```
Comment (kixelated) [inline draft-lcurley-moq-transport.md:170]: What about working top-down right than bottom up?

I think it makes more sense to describe tracks first, and then groups, and then objects, and then anything smaller (application).

**Review by kixelated — COMMENTED** [review]
We spent a while debating the object/group text so I'd prefer smaller changes.

File: draft-lcurley-moq-transport.md (hunk 3 above)
Comment (suhasHere) [reply draft-lcurley-moq-transport.md:170]: totally agree .. This was my suggestion on google docs too . Its more natural that way.

Comment (afrind) [reply draft-lcurley-moq-transport.md:170]: I also had the same suggestion, though I think reading bottom up is still getting the job done.  If we're pressed for time today for -05, I would leave this and revisit it later.

**Review by kixelated — APPROVED** [review]

**Pull request merged**